buildsystem-core/src/main/java/de/eintosti/buildsystem/BuildSystem.java

```diff
@@ -132,6 +132,7 @@ public void onLoad() {
     @Override
     public void onEnable() {
         this.getConfig().options().copyDefaults(true);
+        this.saveResource("messages.yml", false);
```

Comment:
This method needs a `"messages.yml"` embedded in the plugin jar, but the messages are inside `Messages.java`

Reply:
I will fix it tomorrow